tox.ini

```
@@ -0,0 +1,32 @@
+[tox]
+skipsdist=True
+envlist=pycodestyle,pylint,yamllint
+
+[testenv]
+deps =
+  -cconstraints.txt
+  -raws/requirements.txt
+
+[testenv:pylint]
+deps =
+  {[testenv]deps}
+  pylint
+commands = pylint --rcfile hacking/pylint.rc aws
+
+[testenv:pycodestyle]
+deps =
+  {[testenv]deps}
+  pycodestyle
+commands = pycodestyle --config pycodestyle.ini aws
+
+[testenv:yamllint]
+deps =
+  {[testenv]deps}
+  yamllint
+commands = yamllint --config-file .yamllint aws hacking
+
+[testenv:policy]
+deps =
+  {[testenv]deps}
+  ansible-core
+commands = ansible-playbook -i localhost hacking/aws_config/test-policies.yml
```

I'd use full paths where possible:
```suggestion
commands = ansible-playbook -i localhost {toxinidir}/hacking/aws_config/test-policies.yml
```